pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/metrics/xml/CyclomaticComplexity.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data>
+<!--
```

Comment:
That was a lapse, I commented them to ease debugging but they should work. I'll correct that